config/docs/module.yaml

```diff
@@ -15,6 +15,8 @@ mounts:
       - js/index.js
   - source: node_modules/font-awesome/scss
     target: assets/scss/font-awesome
+  - source: node_modules/font-awesome/css
```

Comment:
Please undo this line duplication

Reply:
Hi @XavierChanth , how can I have font-awesome/scss and /css to target assets/scss/font-awesome?
Deleting lines 18 and 19 causes the info icon to stop rendering

Reply:
Purging and reinstalling the modules worked.